tests/unit/snow/test_databricks.py

```diff
@@ -5,17 +5,29 @@
 # pylint: disable=too-many-lines
 import pytest
 from sqlglot import ParseError, UnsupportedError
+from tests.resources.functional.snowflake.helpers.load_tests import source_and_databricks_sql
 
 
-def test_struct(dialect_context):
+@pytest.mark.parametrize("databricks_sql, source", source_and_databricks_sql("test_struct.sql"))
+def test_struct(dialect_context, databricks_sql, source):
     validate_source_transpile, _ = dialect_context
-    # Test case to validate JSON to STRUCT conversion
-    validate_source_transpile(
-        databricks_sql="""SELECT STRUCT(1 AS a, 2 AS b), ARRAY(STRUCT(11 AS C, 22 AS d), 3)""",
-        source={
-            "snowflake": """SELECT {'a': 1, 'b': 2}, [{'c': 11, 'd': 22}, 3];""",
-        },
-    )
+    validate_source_transpile(databricks_sql=databricks_sql, source={"snowflake": source})
+
+
+@pytest.mark.parametrize("databricks_sql, source",
+                         source_and_databricks_sql(
```

Comment:
it will read from file there will be only one function and which will generate multiple pytest and each test will be name of the file.

Reply:
Done